apps/server/src/modules/user/legacy/repo/helper/aggregation-helper.ts

```diff
@@ -1,7 +1,9 @@
-const { ObjectId } = require('mongoose').Types;
+/* eslint-disable */
```

Comment:
Please also add a comment here like "The code in this file is copied from the legacy part of the server. As it does not meet our ESLint rules, ESLint is disabled for the whole file."